packages/service-provider-server/src/cli-service-provider.ts

```diff
@@ -264,6 +286,12 @@ class CliServiceProvider
   public baseCmdOptions: OperationOptions; // public for testing
   private bus: MongoshBus;
 
+  /**
+   * Stores the most recent topology description from the server's SDAM events:
```

Comment:
This doc comment is the same as the data-service one and seems to imply the same behavior, but it's not; this one only returns the last topology at the time when `.connect()` finishes.

I think it's worth either adjusting the doc comment, or, preferably, actually keeping this value up to date beyond the `CliServiceProvider` instantiation